api/api.go

```diff
@@ -107,6 +148,12 @@ func (api *ImportAPI) getJobs(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	setJSONContentType(w)
+
+	if err := api.auditor.Record(r.Context(), getJobsAuditAction, actionSuccessful, nil); err != nil {
+		log.ErrorC("error while attempting to audit event, failing request", err, nil)
```

Comment:
Consider making this 'audit failed' error event string a common string and reusing it throughout - the one above for getJobs and below for getJob are all slightly different so would be difficult to consistently find all audit failures in log aggregation. the action should be in the error already (or better yet we could add it to log.Data)